lib/inv_multi_cg_quda.cpp

```diff
@@ -509,11 +509,9 @@ namespace quda {
 
       for (int i = 0; i < num_offset; i++) {
         // only calculate true residual if we need to:
-        // 1.) for higher shifts if requested tolerance is larger than
-        // the precision of the sloppy solve, i.e. we might not need
-        // to refine
-        // 2.) we did not exit early and shift is 0
-        if ( (i>0 and not mixed) or (i == 0 and not exit_early) ) {
+        // 1.) For shift 0 if we did not exit early  (we went to the full solution)
```

Comment:
@mathiaswagner comments got switched around here to not match the order of the if statement

Reply:
fixed.